exec/java-exec/src/main/resources/rest/options.ftl

```diff
@@ -24,8 +24,24 @@
     <script>
     //Alter System Values
     function alterSysOption(optionName, optionValue, optionKind) {
+        var currHref = location.href;
+        var redirectHref = currHref.replace(/(.?filter=).*/,"");
+        //Read filter value and apply to reload with new filter
+        var reApplyFilter = $("#searchBox").val();
+        if (reApplyFilter != null && reApplyFilter.trim().length > 0)  {
+            redirectHref = redirectHref + "?filter=" + reApplyFilter.trim();
+        } else { //Apply filter for updated field
+            redirectHref = redirectHref + "?filter=" + optionName;
+        }
+        //Debug
+        console.log("Current: "+currHref);
```

Comment:
This will output message into the console each time method is called. Please remove.